iroh-ctl/src/main.rs

```diff
@@ -67,6 +90,8 @@ async fn main() -> anyhow::Result<()> {
     )
     .unwrap();
 
+    iroh_metrics::init_tracer(MetricsConfig::default()).expect("failed to initialize logging");
```

Comment:
@Arqu pulled this from `iroh_metrics`, is this what you were referring to?

Reply:
Sorry for the lag, days are a bit smushed. Just took a proper look and you we're right with just instantiating the handle. This used to be the case before the last refactor but now metrics and tracing are default off and it only sets up your logger unless you pass in flags.

Can we revert to `let metrics_handle = iroh_metrics::MetricsHandle::new(MetricsConfig::default())` that should be good.

Reply:
will do! Really appreciate you taking the time to work through that.